DevIncubator-Autopark/Engine.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace DevIncubatorAutopark
+{
+    internal class Engine
+    {
+		public string TypeName { get; }
+		public double TaxCoefficient { get; }
+
+		public Engine(string typeName, double taxCoefficient)
+		{
+			TypeName = typeName;
+			TaxCoefficient = taxCoefficient;
+		}
+	}
```

Comment:
Formatting. To prevent formatting from breaking, you can replace all tabs with spaces. It helped me

Reply:
fixed